packages/cli/skills/pikku-emails/SKILL.md

```
@@ -0,0 +1,157 @@
+// pikku.config.json
+{
+  "emailTemplatesDir": "emails",   // relative to rootDir; omit to disable emails
+  "outDir": ".pikku"               // gen lands in <outDir>/email/
+}
+```
+
+If `emailTemplatesDir` is unset the command is a no-op.
+
+## Directory layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)